app/kumactl/cmd/get/get_traffic_logs_test.go

```diff
@@ -173,6 +179,57 @@ var _ = Describe("kumactl get traffic-logs", func() {
 				matcher:      MatchYAML,
 			}),
 		)
+
+		DescribeTable("kumactl get traffic-logs web2-to-backend2.namespace2  -o table|json|yaml",
+			func(given testCase) {
+				// given
+
+				args := append([]string{
+					"--config-file", filepath.Join("..", "testdata", "sample-kumactl.config.yaml"),
+					"get", "traffic-logs", "web2-to-backend2.namespace2"})
```

Comment:
This is a test case for `kubernetes` environment (because of `web2-to-backend2.namespace2`).

We also need a test case for `universal` environment (i.e. `web2-to-backend2`)